File: app/src/main/java/info/nightscout/androidaps/plugins/constraints/objectives/objectives/Objective8.java
package info.nightscout.androidaps.plugins.constraints.objectives.objectives;

import java.util.List;

import info.nightscout.androidaps.R;
import info.nightscout.androidaps.plugins.constraints.objectives.ObjectivesPlugin;
import info.nightscout.androidaps.utils.T;

public class Objective8 extends Objective {

    public Objective8() {
        super("ama", R.string.objectives_ama_objective, 0);
    }

    @Override
    protected void setupTasks(List<Task> tasks) {
        tasks.add(new MinimumDurationTask(T.days(28).msecs()));
    }
}

File: app/src/main/java/info/nightscout/androidaps/plugins/constraints/objectives/objectives/Objective8.kt
package info.nightscout.androidaps.plugins.constraints.objectives.objectives

import dagger.android.HasAndroidInjector
import info.nightscout.androidaps.R
import info.nightscout.androidaps.utils.T

class Objective8(injector: HasAndroidInjector) : Objective(injector, "ama", R.string.objectives_ama_objective, 0) {

    init {
        tasks.add(MinimumDurationTask(this, T.days(28).msecs()))
    }
}